core/blockly.js

```diff
@@ -141,36 +141,30 @@ Blockly.svgSize = function(svg) {
 };
 
 /**
- * Schedule a call to the resize handler.  Groups of simultaneous events (e.g.
- * a tree of blocks being deleted) are merged into one call.
- * @param {Blockly.WorkspaceSvg} workspace Any workspace in the SVG.
+ * Size the workspace when the contents change.  This also updates
+ * scrollbars accordingly.
+ * @param {!Blockly.WorkspaceSvg} opt_workspace. The svg group of the workspace
+ *   to resize. If the workspace isn't specified, the main workspace will be
+ *   used.
  */
-Blockly.asyncSvgResize = function(workspace) {
-  if (Blockly.svgResizePending_) {
-    return;
-  }
+Blockly.resizeSvgContents = function(opt_workspace) {
+  var workspace = opt_workspace;
   if (!workspace) {
```

Comment:
Thanks for pointing this out.  I had kept the if (!workspace) there because asyncResize had it and that's essentially what I replaced.  I believe the reason asyncResize had it is because when it was called from Blockly.BlockSvg.terminateDrag_, the workspace is undefined. It passes along anyway, but it turns out, the workspace is undefined since Blockly.BlockSvg.terminateDrag is not defined on the prototype and that is presumably because the call chain is through Blockly.terminateDrag which doesn't have a workspace.  For example, the call chain when a mouse up happens is:

Blockly.BlockSvg.onMouseUp_  (has a workspace reference)
Blockly.terminateDrag_ (doesn't have workspace reference)
Blockly.BlockSvg.terminateDrag_ (doesn't have workspace reference but looks like it does b/c its in BlockSvg)
Blockly.resizeSvgContents


So then I think options are:
1. leave in the if(!workspace) check in resizeSvgContents and remove the parameter passing from BlockSvg.terminateDrag_ so it doesn't look like its doing something it isn't.
2. pass the workspace through terminateDrag?
3. other thoughts?